config/Dockerfiles/tests.d/aws/02_aws_ec2_vpc_net

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -xe
 
 # Verify the aws ec2 vpc net provisioning
-# distros.exclude: centos7 fedora28
+# distros.exclude: centos7 fedora27 fedora28
```

Comment:
Shouldn't this be 28 and 29 now?

Reply:
Probably needs to be rebased?

Reply:
It should be just 28 for now since I would like to test it on f29 only.